docs/cloud-adoption/infrastructure/overview.md

```diff
@@ -5,93 +5,85 @@ author: rotycenh
 ms.date: 11/02/2018
 ---
 
-# Fusion: Core Infrastructure overview
+# Fusion: Core infrastructure overview
 
-One of the most important steps your organization takes on its digital
-transformation journey is understanding how cloud platforms work. Knowing how
-resources are deployed, organized, managed, and brought into compliance with
-your overall governance policies is critical in making the correct architectural
-decisions for your migration.
+Understanding how cloud platforms work is about knowing how resources are deployed, 
+organized, managed, and brought into compliance with your overall governance policies. 
+This first step is critical in making correct architectural decisions for your migration.  
 
-This section of the Fusion guidance will provide a summary of
-the most important technologies and models used in designing and building
-cloud-based solutions, as well as providing some concrete examples and
-prescriptive guidance on building solutions specifically using the Azure public
-cloud platform.
+This section provides a summary of the most important technologies and models 
+used in designing and building cloud-based solutions, as well as providing concrete 
+examples and guidance for building solutions, specifically those that use the Azure 
+public cloud platform.
 
 ## Core infrastructure components
 
-Each of the following components represents one of the key foundational
-technologies underlying all cloud deployments. It's important to learn how these
-components work and which of their various features and deployment models are
-appropriate for your needs.
+Each of the following components represents a key foundational technology underlying 
+all cloud deployments. It's important to learn how these components work, and to 
+understand which features and deployment models are appropriate for your needs.
 
 [Subscriptions](subscriptions/overview.md):
-At the root of your cloud deployment's ownership, billing, and management
-capabilities is an underlying account or subscription.
+Configure your Azure subscription. This is the account for defining your cloud deployment's ownership, billing, 
```

Comment:
This is not technically correct. In Azure a subscription is distinct from an account, although there isn't always the same distinction in other cloud services. Note also that this section is not meant to be exclusively focused on Azure  but discuss cloud platform architecture in a more general way when possible (although the subscriptions topic kind of does focus on structuring your enterprise agreement).